gridstatus/isone_api/isone_api.py

```diff
@@ -355,4 +377,8 @@ def get_dayahead_hourly_demand(
             )
 
         df = pd.DataFrame(all_data)
+        # NOTE(kladar): 2017-07-01 to 2018-06-01 causes an issue
+        # as there are duplicates of the .H.INTERNALHUB location. Deduping them here
+        df = df.drop_duplicates(subset=["BeginDate", "Location"], keep="first")
```

Comment:
@WillKoehrsen added this to de-dupe the verified duplicates and any more that may appear. Could make sense to move to `_handle_demand` but don't want to belabor this too much.

Reply:
Dropping duplicates is fine if we verified the duplicates occur in the source data (and aren't an artifact of our data).